department-of-veterans-affairs/abd-vro
# MCP-2437: Resolve checkstyle warnings

## What was the problem?

Checkstyle warnings from improper formatting. 
Associated tickets or Slack threads:

- [MCP-2437](https://amida.atlassian.net/browse/MCP-2437)

## How does this fix it?[^1]

Fix formatting where there are checkstyle warnings.
## How to test this PR
- Step 1 run ./gradlew check
- Make sure there are no more warnings
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
